icu4c/source/test/cintltst/ccaltst.c

```diff
@@ -2553,4 +2555,221 @@ void TestUcalOpenBufferRead() {
     ucal_close(cal);
 }
 
+
+#define NUM_DATES 10
```

Comment:
Why not use something like UPRV_LENGTHOF(array)?

Reply:
done